examples/quantization_aware_training/setup.py

```diff
@@ -0,0 +1,8 @@
+from setuptools import setup, find_packages
```

Comment:
Is this setup needed, or is this handled by torch2trt setup with ``--experimental`` flag?